SoftLayer/CLI/virt/create.py

```diff
@@ -175,6 +176,9 @@ def _parse_create_args(client, args):
               help="OS install code. Tip: you can specify <OS>_LATEST")
 @click.option('--image',
               help="Image ID. See: 'slcli image list' for reference")
+@click.option('--boot-mode',
+              help="Specify the mode to boot the OS in",
```

Comment:
some context here on what boot modes are available would be good. 
Updating https://softlayer-python.readthedocs.io/en/latest/cli/vs.html (https://github.com/softlayer/softlayer-python/blob/master/docs/cli/vs.rst ) on what choosing a different boot mode does would also be helpful

Reply:
I'll add the boot modes here and get with offering for documentation to add to https://softlayer-python.readthedocs.io/en/latest/cli/vs.html.